### DISCUSSION TITLE
UI Kit 3.0.1 Purchase Includes One Year of Free Updates

### DISCUSSION THREAD

**Discussion by irekhviashvili** [primefaces/community#3280] [Figma UI Kit] [answered]
As mentioned on PrimeNG website, the UI kit purchase includes one year of free updates. However, I noticed that the updates for the PrimeNG v19 were not included with the kit.

Could you please clarify why the UI Suite updates were not included with the UI kit I purchased? Any guidance or explanation regarding this matter would be greatly appreciated.

**Comment by onursenture** [comment]
Could you please elaborate on what you mean by "the updates for the PrimeNG v19 were not included"? The UI Kit updates may not be released at the same time as the PrimeNG/PrimeVue/PrimeReact releases, and we have not made any promises regarding this. 

However, if there is something present in PrimeNG 19 that you find missing in the latest version of the UI Kit, we would like to add it to our list for inclusion in a future update.

**Reply by irekhviashvili** [reply]
In UI Kit v3.0.1, the colors in some cases do not match the colors of the components in PrimeNG 17 (What we're using at this moment). For example, the color of the Success tag is different in the UI Kit compared to PrimeNG. This is the issue I am facing and the reason of my discussion topic.

**Reply by onursenture** [reply, marked as answer]
It is quite likely that the colors in UI 3.0.1 are incompatible with PrimeNG 17, as 3.0.1 was designed to be compatible with the latest version of NG.

 If you experience incompatibility with the latest version, please report it so we can note it for correction in future versions.
